src/languages/elixir.md

```diff
@@ -0,0 +1,135 @@
+# Elixir
+
+Platform.sh supports building and deploying applications written in Elixir.  They are automatically compiled during the Build phase, and support both committed dependencies and download-on-demand. The underlying Erlang version is 
+22.0.7.
+
+## Supported versions
+
+* 1.9
```

Comment:
Didn't the previous paragraph just say 22.0.7?

Reply:
22.0.7 is the version of Erlang; 1.9 is the version of Elixir, which runs on the Erlang VM so has a dependency on it.